### PR TITLE
fix: tiktok add missing field brand

### PR DESCRIPTION
**Pull request by anantjain45823**
## PR Description
Resolves INT-1182
Currently we are not mapping `brand` for each of `properties.products.$.brand` or `properties.brand` getting mapped to `properties.contents.$.brand`.
Since tiktok support brand as contents.$.brand we need to map it as well.
This PR introduces this mapping

## Linear task (optional)

Linear task link

## Cross Browser Tests

Please confirm you have tested for the following browsers:

- [ ] Chrome
- [ ] Firefox
- [ ] IE11

## Sanity Suite

- [ ] All sanity suite test cases pass locally

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

File: packages/analytics-js-integrations/__tests__/integrations/TikTokAds/tiktokads.test.js [hunk 1]
```diff
@@ -375,6 +378,7 @@ describe('TiktokAds Track event', () => {
       partner_name: 'RudderStack',
       contents: [
         {
+          brand: '',
```
Comment (koladilip) [inline packages/analytics-js-integrations/__tests__/integrations/TikTokAds/tiktokads.test.js:381]: why blank?

Comment (anantjain45823) [reply packages/analytics-js-integrations/__tests__/integrations/TikTokAds/tiktokads.test.js:381]: not any particular reason just changed the output based upon input vals

Comment (koladilip) [reply packages/analytics-js-integrations/__tests__/integrations/TikTokAds/tiktokads.test.js:381]: Better to choose non blank value

File: packages/analytics-js-integrations/__tests__/integrations/TikTokAds/tiktokads.test.js [hunk 2]
```diff
@@ -186,6 +186,7 @@ describe('TiktokAds Track event', () => {
           content_type: 'CONTENT_TYPE',
           price: 50.0,
           quantity: 1,
+          brand: '',
```
Comment (yashasvibajpai) [inline packages/analytics-js-integrations/__tests__/integrations/TikTokAds/tiktokads.test.js:189]: no data coming in brand, can you pls update the test input to get some valid data here?

Comment (anantjain45823) [reply packages/analytics-js-integrations/__tests__/integrations/TikTokAds/tiktokads.test.js]: Done

**Review by koladilip — APPROVED** [review]

**Review by yashasvibajpai — APPROVED** [review]

**Pull request merged**